### PR TITLE
Code Tab added for footer component

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

File: website/screens/components/footer/usage/examples/Icons.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import React from "react";
```
Comment (raquelarrojo) [inline website/screens/components/footer/usage/examples/Icons.jsx:1]: Rename to icons.tsx.
Delete unnecessary import.

File: website/screens/components/footer/code/examples/Icons.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import React from "react";
```
Comment (raquelarrojo) [inline website/screens/components/footer/code/examples/Icons.jsx:1]: Remove file, as it is not needed for basic usage example.

**Review by raquelarrojo — CHANGES_REQUESTED** [review]

File: website/screens/components/footer/usage/FooterUsagePage.tsx [hunk 3]
```diff
@@ -6,15 +6,16 @@ import DocFooter from "@/common/DocFooter";
 import Figure from "@/common/Figure";
 import QuickNavContainer from "@/common/QuickNavContainer";
 import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Example from "@/common/example/Example";
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/footer/usage/FooterUsagePage.tsx:5]: Done.

File: website/screens/components/footer/code/examples/Basic.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import { DxcFooter } from "@dxc-technology/halstack-react";
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/footer/code/examples/Basic.ts]: Done

File: website/screens/components/footer/code/examples/Basic.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      text: "Facebook",
+    },
+  ];
+  return (
+    <DxcFooter
+      bottomLinks={bottom}
+      socialLinks={social}
+    >
+    </DxcFooter>
+  );
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/footer/code/examples/Basic.ts]: Done

File: website/screens/components/footer/usage/examples/ContentFooter.ts [hunk 6]
```diff
@@ -0,0 +1,172 @@
+import { DxcFooter, DxcRow, DxcStack } from "@dxc-technology/halstack-react";
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/footer/usage/examples/ContentFooter.ts]: Done

File: website/screens/components/footer/usage/examples/UsageFooter.ts [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+      text: "Feedback",
+    },
+  ];
+  return (
+    <DxcFooter
+      bottomLinks={bottom}
+      socialLinks={social}
+      copyright="© DXC Technology Company"
+    >
+    </DxcFooter>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/footer/usage/examples/UsageFooter.ts]: Done

File: website/screens/components/footer/code/examples/Icons.jsx (hunk 2 above)
Comment (Lisa159) [reply website/screens/components/footer/code/examples/Icons.jsx]: Done

File: website/screens/components/footer/usage/examples/UsageFooter.ts [hunk 8]
```diff
@@ -0,0 +1,50 @@
+import { DxcFooter } from "@dxc-technology/halstack-react";
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/footer/usage/examples/UsageFooter.ts]: Done

File: website/screens/components/footer/usage/examples/Icons.jsx (hunk 1 above)
Comment (Lisa159) [reply website/screens/components/footer/usage/examples/Icons.jsx]: As for .ts files are in lowercase and .tsx files are with capital letters, I will keep this file in capital letters.
Changed to .tsx.

File: website/screens/components/footer/usage/examples/ContentFooter.ts [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+              }}
+            >
+              Life science
+            </div>
+          </DxcStack>
+        </DxcRow>
+      </div>
+    </DxcFooter>
+  );
+}`;
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/footer/usage/examples/ContentFooter.ts]: If div is changed to DxcInset, the inline style will not be applying (color will be black and you will not be able to read the word).
However, I changed the first div (the one with marginTop) to DxcInset.

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**